engine/src/main/java/io/zeebe/engine/processor/workflow/job/JobBatchActivateProcessor.java

```diff
@@ -221,4 +236,31 @@ private void rejectCommand(
     streamWriter.appendRejection(record, rejectionType, rejectionReason);
     responseWriter.writeRejectionOnCommand(record, rejectionType, rejectionReason);
   }
+
+  private void raiseIncidentJobTooLargeForMessageSize(
+      long jobKey, final JobRecord job, final TypedStreamWriter streamWriter) {
+
+    final DirectBuffer jobErrorMessage = job.getErrorMessageBuffer();
+    DirectBuffer incidentErrorMessage =
+        wrapString(
+            String.format(
+                "The job with key '%s' is too large for the configured message size.", jobKey));
+    if (jobErrorMessage.capacity() > 0) {
+      incidentErrorMessage = jobErrorMessage;
+    }
+
+    final IncidentRecord incidentEvent =
+        new IncidentRecord()
+            .setErrorType(ErrorType.CONDITION_ERROR.IO_MAPPING_ERROR)
```

Comment:
We need a new error type for this incident. The existing types don't fit.

Reply:
@saig0 Is this a problem with backwards compatibility? Or is it just important to add the new error code at the end?

Reply:
Good point. We should add it at the end to avoid any problem on deserialization.